### ISSUE TITLE
Traversing through slides caused the 503 Service Unavailable Error to appear.

### ISSUE THREAD

**Issue by RohanAsmat** [slidewiki/slidewiki-platform#604]
![screenshot from 2017-10-18 10-50-28](https://user-images.githubusercontent.com/7516846/31709581-b6f6c28c-b3f2-11e7-81bf-362760ea06a7.png)
While traversing through slides, the error shown in the figure appeared. Although the slide navigator to switch back and forth worked perfectly but presentation view of the slide was unable to load the new slides.

**Comment by kadevgraaf** [comment]
Hi @RohanAsmat . Thanks for reporting this!
I think this is due to performance issues on large decks. 
Can you still navigate to next/new slides if you wait for ~20 seconds?

**Comment by RohanAsmat** [comment]
Yes, I found that on Slidewiki.org and I had problem traversing through, but once I refreshed my 
web page twice I was able to easily navigate through slides.
